### DISCUSSION TITLE
No such file or directory when including a CGAL header

### DISCUSSION THREAD

**Discussion by AlbertDeTerre** [CGAL/cgal#8122] [Q&A] [answered]
Hello,

I downloaded cgal in my project by doing this:

``` {.bash}
 git clone https://github.com/CGAL/cgal.git ./libraries/cgal.git
 ./libraries/cgal.git/Scripts/scripts/cgal_create_cmake_script
 mkdir -p build/release
 cd build/release
 cmake -DCMAKE_BUILD_TYPE=Release -DCGAL_DIR:PATH=./libraries/cgal.git ../..
 make
```

But I got a `No such file or directory` when I add ` #include <Exact_predicates_exact_constructions_kernel.h>` to my main.cpp file...

### My CMakeLists.txt file:

```
cmake_minimum_required(VERSION 3.1...3.23)
project( stl-to-primitive )

find_package(CGAL REQUIRED QUIET OPTIONAL_COMPONENTS Core)

add_executable(stl-to-primitive src/main.cpp)
```

**Comment by sloriot** [comment]
should be `CGAL/Exact_predicates_exact_constructions_kernel.h`

**Reply by AlbertDeTerre** [reply]
I tried using this one and it caused the same error

**Reply by sloriot** [reply, marked as answer]
`target_link_libraries(stl-to-primitive CGAL::CGAL)` is also needed in the cmake script.

**Reply by AlbertDeTerre** [reply]
Adding `target_link_libraries(stl-to-primitive CGAL::CGAL)` made it work ! Thanks a lot :)
